src/utils/features.tsx

```diff
@@ -28,10 +28,10 @@ export const featuresCardsData = [
         icon: VSCGrey,
         iconTitle: <span>VS Code<br />Extensions</span>,
         paragraphs: <React.Fragment>
-            <p>Gitpod feels just like your local dev environment, as you can install any VS Code extensions. Install it via drag and drop and enjoy all the features you're used to.</p>
+            <p>Gitpod feels just like your local dev environment, as you can <strong> install any <br></br>VS Code extensions</strong>. Install it via drag and drop and enjoy all the features you're used to.</p>
```

Comment:
Not sure you need the `<br></br>` here.

```suggestion
            <p>Gitpod feels just like your local dev environment, as you can <strong> install any VS Code extensions</strong>. Install it with a drag-and-drop, and enjoy all the features you're used to.</p>
```